packages/react-router/src/useNavigate.tsx

```diff
@@ -29,7 +29,6 @@ export function useNavigate<
     (options: NavigateOptions) => {
       return router.navigate({
         ...options,
-        from: options.to ? router.state.resolvedLocation.pathname : undefined,
```

Comment:
For `useNavigate`, would `from` just serve to provide type-completion then?

Reply:
It's used at runtime as well. Its used to build the location. As long as it partially matches the current location then it can be used